extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -360,6 +360,11 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
         expect(ampAnalyticsElement.innerHTML.indexOf(url)).to.not.equal(-1);
       });
     });
+    it('centers iframe in slot', () => {
```

Comment:
There should also be test cases for the "no height and width are set on the amp-ad", "both height and width are set", and "only one of height/width is set" cases.